docs/deprecated.md

```diff
@@ -21,41 +21,41 @@ To learn more about Docker Engine's deprecation policy,
 see [Feature Deprecation Policy](https://docs.docker.com/engine/#feature-deprecation-policy).
 
 ## `filter` param for `/images/json` endpoint
-**Deprecated In Release: [v1.13](https://github.com/docker/docker/releases/tag/v1.13.0)**
+**Deprecated In Release: [v1.13.0](https://github.com/docker/docker/releases/)**
```

Comment:
I believe this is intended. The hyperlink will be valid when v1.13.0 is released